examples/diffantom/sticks2dwis

```diff
@@ -56,6 +57,8 @@ parser.add_argument('-b', dest='bvals', required=True,
                     help="b-values in a text file.")
 parser.add_argument('-r', dest='bvecs', required=True,
                     help="b-vectors in a text file.")
+parser.add_argument('--b_perps', type=str, default=None,
+                    help="If supplied, b_perpendiculars of b-tensors in a text file.")
```

Comment:
Cool, may I suggest to add an example (bval, bvec and bperp files)? It could be in an example/schemes folder for instance.